docs(chart): Add kb about how to apply gradient color effect to area … (#1257)
telerik/blazor-docs: knowledge-base/chart-series-gradient-color.md

---
title: Apply gradient effect on Chart Series
description: How to apply gradient as color for Chart series?
type: how-to
page_title: Apply gradient effect on Chart series
slug: chart-series-gradient-color
position: 
tags: chart, series, gradient, color
ticketid: 1545645, 1591373
res_type: kb
---

## Environment
<table>
	<tbody>
		<tr>
			<td>Product</td>
			<td>Chart for Blazor</td>
		</tr>
	</tbody>
</table>

## Description
How can I apply a gradient color effect on the Telerik UI for Blazor Chart series?

## Solution

1. Declare an external [linear gradient with an `svg` tag](https://developer.mozilla.org/en-US/docs/Web/SVG/Element/linearGradient).

> 1. Use the [Color](https://docs.telerik.com/blazor-ui/components/chart/types/area#color) parameter of the Chart series to set the external gradient via its `id`. 

Reviewer: ```suggestion
1. Set the [Color](https://docs.telerik.com/blazor-ui/components/chart/types/area#color) parameter of the `ChartSeries` and refer the external gradient via its `id`. 
```
